### ISSUE TITLE
Cannot find module 'ios-sim-portable'

### ISSUE THREAD

**Issue by tchon** [NativeScript/nativescript-cli#245]
When running `tns run ios --emulator` using nativescript 0.4.2 and following hello-world quickstart, the following runtime error is occurring:

Cannot find module 'ios-sim-portable'

**Comment by rosen-vladimirov** [comment]
Hi @tchon 

Are you using tns directly installed from npm or you have cloned the repo and rebuild it on your side? In case you have cloned the repo, did you execute npm install inside the project? The ios-sim-portable is a command-line utility written in Node.js that launches an iOS application bundle (.app) in the Xcode iOS Simulator. It looks like it is missing on your side, so you can try finding where your tns is installed (executing "$ which tns" should help) and executing npm install in the directory. This should install all dependencies described in package.json. Could you give it a try and inform us if it works for you?

**Comment by tchon** [comment]
@rosen-vladimirov I did a `npm install nativescript -g` and it looks like it installs the latest tagged release v0.4.2. After I did a `git checkout` on the nativescript-cli project, I can see now that ios-sim-portable is list5ed in the package.json.  So, it seems the ios-sim-portable dependency was made after the v0.4.2 tag. 

For now, I just did a `npm install ios-sim-portable -g` and that fixed it for me to run the app in emulator mode. I'll try later tonight to see if the latest on master works. Thanks for your help!
